packages/capacitor-plugin/src/definitions.ts

```diff
@@ -0,0 +1,565 @@
+   * It's not accesible on Android 10 unless the app enables legacy External Storage
+   * by adding `android:requestLegacyExternalStorage="true"` in the `application` tag
+   * in the `AndroidManifest.xml`.
+   * It's not accesible on Android 11 or newer.
+   *
+   * @since 1.0.0
+   */
+  ExternalCache = 'EXTERNAL_CACHE',
+
+  /**
+   * iOS only
+   *
+   * @since 1.0.0
+   */
+  LibraryNoCloud = 'LIBRARY_NO_CLOUD',
+
+  /**
+  * iOS only
+  *
+  * @since 1.0.0
+  */
+  Temporary = 'TEMPORARY',
+}
+
+export enum Encoding {
+  /**
+   * Eight-bit UCS Transformation Format
+   *
+   * @since 1.0.0
+   */
+  UTF8 = 'utf8',
+
+  /**
+   * Seven-bit ASCII, a.k.a. ISO646-US, a.k.a. the Basic Latin block of the
+   * Unicode character set
+   * This encoding is only supported on Android.
+   *
+   * @since 1.0.0
+   */
+  ASCII = 'ascii',
+
+  /**
+   * Sixteen-bit UCS Transformation Format, byte order identified by an
+   * optional byte-order mark
+   * This encoding is only supported on Android.
+   *
+   * @since 1.0.0
+   */
+  UTF16 = 'utf16',
+}
+
+export interface WriteFileOptions {
+  /**
+   * The path of the file to write
+   *
+   * @since 1.0.0
+   */
+  path: string;
+
+  /**
+   * The data to write
+   *
+   * Note: Blob data is only supported on Web.
+   *
+   * @since 1.0.0
+   */
+  data: string | Blob;
+
+  /**
+   * The `Directory` to store the file in
+   *
+   * @since 1.0.0
+   */
+  directory?: Directory;
+
+  /**
+   * The encoding to write the file in. If not provided, data
+   * is written as base64 encoded.
+   *
+   * Pass Encoding.UTF8 to write data as string
+   *
+   * @since 1.0.0
+   */
+  encoding?: Encoding;
+
+  /**
+   * Whether to create any missing parent directories.
+   *
+   * @default false
+   * @since 1.0.0
+   */
+  recursive?: boolean;
+}
+
+export interface AppendFileOptions {
+  /**
+   * The path of the file to append
+   *
+   * @since 1.0.0
+   */
+  path: string;
+
+  /**
+   * The data to write
+   *
+   * @since 1.0.0
+   */
+  data: string;
+
+  /**
+   * The `Directory` to store the file in
+   *
+   * @since 1.0.0
+   */
+  directory?: Directory;
+
+  /**
+   * The encoding to write the file in. If not provided, data
+   * is written as base64 encoded.
+   *
+   * Pass Encoding.UTF8 to write data as string
+   *
+   * @since 1.0.0
+   */
+  encoding?: Encoding;
+}
+
+export interface ReadFileOptions {
+  /**
+   * The path of the file to read
+   *
+   * @since 1.0.0
+   */
+  path: string;
+
+  /**
+   * The `Directory` to read the file from
+   *
+   * @since 1.0.0
+   */
+  directory?: Directory;
+
+  /**
+   * The encoding to read the file in, if not provided, data
+   * is read as binary and returned as base64 encoded.
+   *
+   * Pass Encoding.UTF8 to read data as string
+   *
+   * @since 1.0.0
+   */
+  encoding?: Encoding;
+}
+
+export interface ReadFileInChunksOptions extends ReadFileOptions {
+  /**
+   * Size of the chunks in bytes.
+   *
+   * @since 1.0.0
```

Comment:
This is only offered in the future plugin version, which I think will be

```suggestion
   * @since 7.1.0
```

Reply:
fixed